### PR TITLE
Feature/dynamic iptables

### PR DESCRIPTION
**Pull request by langston-barrett**
Make this role an addon of mantl, rather than a core component. Will merge to master once this is merged.

### CODE REVIEW

File: addons/iptables.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
```
Comment (eirslett) [inline addons/iptables.yml:1]: Is it possible to import this playbook snippet, "requirements.yml" or something? Then, every other Mantl playbook could depend on it.

Comment (langston-barrett) [inline addons/iptables.yml:1]: Do you mean like #1024?

Comment (eirslett) [inline addons/iptables.yml:1]: yes, exactly!

Comment (eirslett) [inline addons/iptables.yml:1]: But I'll merge this, and then that refactoring can take place in the mantl project whenever it's ready!

Comment (langston-barrett) [inline addons/iptables.yml:1]: PR #1040 will address that issue. If it gets merged before this one, we should definitely use an `include`. Good thoughts as always @eirslett :)

**Pull request merged**